### PR TITLE
Remove alembic stuff

### PR DESCRIPTION
**Pull request by albertkol**
## Done

- Remove `alembic` stuff. The API will take care of it.

## QA

- Are checks passing?

## Issue / Card

Part of #9017

### CODE REVIEW

File: entrypoint [hunk 1]
```diff
@@ -5,24 +5,7 @@ set -e
 RUN_COMMAND="talisker.gunicorn.gevent webapp.app:app --bind $1 --worker-class gevent --name talisker-`hostname`"
 
 if [ "${FLASK_DEBUG}" = true ] || [ "${FLASK_DEBUG}" = 1 ]; then
```
Comment (jpmartinspt) [inline entrypoint:7]: This needs to be an if else.

Comment (albertkol) [reply entrypoint:7]: Not sure I understand. Isn't this what we use for most of our websites? Fx:
https://github.com/canonical-web-and-design/jp.ubuntu.com/blob/master/entrypoint
https://github.com/canonical-web-and-design/charmed-kubeflow.io/blob/master/entrypoint

Comment (jpmartinspt) [reply entrypoint:7]: NVM, I was looking at the diff and did not realise you swapped the logic inside the if to be an assignment.

**Review by jpmartinspt — APPROVED** [review]

**Comment by jpmartinspt** [comment]
Looks good, needs a rebase before merging though.

**Comment by carkod** [comment]
https://ubuntu-com-9106.demos.haus/security/cve seems to be broken

**Comment by jpmartinspt** [comment]
@carkod, since we are removing migrations from ubuntu.com there is no schema created on the demo database. That is why you see the error on `/cve`. The way to QA this would be locally against a database that already has the tables in place.

**Comment by albertkol** [comment]
Closed. For now we will keep the migrations on ubuntu.com. 

We should do this at the very end when ubuntu-com is not reliant on the database anymore. 

That way we avoid any 500s caused by the missing database tables.

**Pull request closed without merging**